### PR TITLE
Swadesh date fix

### PR DESCRIPTION
**Pull request by chrzyki**
Changed date of Russian Swadesh-215 list to 1960. Addresses #531.

### CODE REVIEW

File: concepticondata/conceptlists.tsv [hunk 1]
```diff
@@ -18,8 +18,8 @@ Kingsada-1999-303	Kingsada, Thongpheth and Shintani, Tadahiko	1999		303	basic	En
...
+Swadesh-1960-215	Swadesh, Morris	1964		215	basic	Russian,English	Global	http://www.classes.ru/grammar/148.new-in-linguistics-1/source/worddocuments/_24.htm	Swadesh1964		This is the first Russian translation of the master list of [Swadesh's paper from 1955](:bib:Swadesh1955), which turned out to be very influential in Soviet linguistics and has retained influential up to today. Note that while the translation of the concept labels is literal and also provides the original ones which Swadesh provided in his c[master list of 215 concepts](:ref:Swadesh-1955-215), the Russian translation makes certain distinctions which cannot be determined from the original text. In one case, in the case of “fat”, this distinction is essentially wrong, as Swadesh did explicitly annotate this concept as a body part or substance. The concepts whose English labels are preceded by an asterisk are those 92 concepts which were later included in the famous [100-item list](:ref:Swadesh-1955-100).	78-87
```
Comment (xrotwang) [inline concepticondata/conceptlists.tsv]: Shouldn't the changed year carry over to the YEAR column as well?

Comment (chrzyki) [reply concepticondata/conceptlists.tsv]: You are right, thanks!

**Comment by chrzyki** [comment]
Thanks & updated.

**Comment by chrzyki** [comment]
Failing tests shouldn't be a problem after merging this, right? Presumably due to the new metadata checks.

**Comment by LinguList** [comment]
I don't understand this completely, but the metadata-issues look like they can be quickly changed by deleting a wrong entry in metadata?

**Comment by chrzyki** [comment]
I think this will be fine after it has been merged - @xrotwang implemented a new check for metadata consistency which is already being applied: https://github.com/concepticon/pyconcepticon/commit/fe17e52ba3e47b2a5fb6424dbdb431f228f2f1d4

This PR lacks the updated metadata lists from the other PR so it should be fine after merge.

**Comment by chrzyki** [comment]
If the date update is okay I'll merge and the build problems should be solved. :)

**Comment by LinguList** [comment]
Okay, fine by me.

**Comment by chrzyki** [comment]
:+1: https://travis-ci.org/clld/concepticon-data

**Pull request merged**